pkgs/tools/security/bloodhound-py/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, fetchFromGitHub
+, python3
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "bloodhound-py";
+  version = "1.0.1";
```

Comment:
The author hasn't tagged it on the VCS, but they have released 1.6.1.
1.0.1 is an older version, over three years ago.